gitapp/lib/users_list/users_list.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:gitapp/users_list/user.dart';
+//Lists of the users FGA-3
+class UsersList {
+  List<User> userlist= List.empty(growable: true);
+  List<User> searchedlist= List.empty(growable: true);
+  UsersList(){
+    for(int i=0;i<5;i++)
```

Comment:
please do not use this for method.
Use map, foreach or something similar. Let's work with higher order functions directly on collections.

Reply:
solved